tools/junitreport/junitreport.go

```diff
@@ -117,13 +120,17 @@ func main() {
 	arguments := flag.Args()
 	// If we are asked to summarize an XML file, that is all we do
 	if len(arguments) == 1 && arguments[0] == "summarize" {
-		summary, err := cmd.Summarize(input)
+		summary, err, ok := cmd.Summarize(input)
 		if err != nil {
 			fmt.Fprintf(os.Stderr, "Error summarizing jUnit XML file: %v\n", err)
 			os.Exit(1)
 		}
 		fmt.Fprint(os.Stdout, summary)
-		os.Exit(0)
+		if ok {
+			os.Exit(0)
+		} else {
+			os.Exit(3)
```

Comment:
This seems weird.  We have no trouble summarizing the results (running this command), but it returns no-zero?  Is there something else you can trigger on?